src/services/UserInfoRequestProcessor.ts

```diff
@@ -50,7 +51,7 @@ export class UserInfoRequestProcessor {
 		this.metrics = metrics;
 		this.cicService = CicService.getInstance(SESSION_TABLE, this.logger, createDynamoDbClient());
 		this.kmsJwtAdapter = new KmsJwtAdapter(KMS_KEY_ARN);
-		this.verifiableCredentialService = VerifiableCredentialService.getInstance(SESSION_TABLE, this.kmsJwtAdapter, ISSUER, this.logger);
+		this.verifiableCredentialService = VerifiableCredentialService.getInstance(SESSION_TABLE, this.kmsJwtAdapter, ISSUER, this.logger, DNS_SUFFIX);
```

Comment:
Could we please add a check for `DNS_SUFFIX` along with the other env vars on line 43?

Reply:
And can we update the corresponding log messge?